apiary.apib

```diff
@@ -3055,6 +3057,8 @@ Get details for a single project.
                 + `provided` (Money)
                 + `spent` (Money)
                 + `remaining` (Money) - The actual budget remaining, not taking the planned work into account.
+                + `allocated` (Money, optional) - The estimated budget for planned work that is still remaining.
```

Comment:
I find these descriptions a bit difficult to understand, can we check with the PO for a better description?